vortex-expr/src/pruning/pruning_expr.rs

```diff
@@ -60,41 +70,29 @@ pub fn field_path_stat_field_name(field_path: &FieldPath, stat: Stat) -> FieldNa
         .into()
 }
 
-/// Create a stat based falsification expr assuming all stats for all column in the expression
-/// exist
-pub fn pruning_expr(expr: &ExprRef) -> Option<(ExprRef, RequiredStats)> {
-    let mut catalog = AnyStatsCatalog {
-        ..Default::default()
-    };
-    let expr = expr.stat_falsification(&mut catalog)?;
-
-    // TODO(joe): filter access by used exprs
-    let mut relation: Relation<FieldPath, Stat> = Relation::new();
-    for ((field_path, stat), _) in catalog.usage.into_iter() {
-        relation.insert(field_path, stat)
-    }
-
-    Some((expr, relation))
-}
```

Comment:
this was only being used in the tests below, I don't think there's a use for a version of this function that doesn't check for stat availability so I just removed this